### PR TITLE
Switchboard support

### PR DESCRIPTION
**Pull request by khyberspache**

### CODE REVIEW

File: plugins/cloud/Cloud.html [hunk 1]
```diff
@@ -229,11 +229,14 @@
         function tunnelSuccess(d){
             $("#cloud-div .loader").removeClass("loading")
             const seconds = 30;
-            const redirector = {
+            let redirector = {
                 user: 'ubuntu',
                 key: config.aws.key_name,
                 availableAt: new Date(Date.now() + (1000 * seconds))
             }
+            if (SETTINGS.account.license === 'professional') {
```
Comment (privateducky) [inline plugins/cloud/Cloud.html:237]: i get that the password is only useful for pro licenses.. but any harm if we associate it regardless (both here and below)? as in, can we just remove the if-clause entirely without impact?

Comment (privateducky) [reply plugins/cloud/Cloud.html:237]: @khyberspache bump on this comment

Comment (khyberspache) [reply plugins/cloud/Cloud.html:237]: Yes this needs to be there

Comment (khyberspache) [reply plugins/cloud/Cloud.html:237]: We use it as a way to distinguish how to connect to the redirector.  Password == use switchboard

**Pull request merged**